proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -35,20 +35,36 @@ service Query {
         "/interchain_security/ccv/provider/consumer_chains";
   }
 
+  // [DEPRECATED]: Use `QueryConsumersThatAreAboutToStart` instead
   // QueryConsumerChainStarts queries consumer chain start proposals.
   rpc QueryConsumerChainStarts(QueryConsumerChainStartProposalsRequest)
       returns (QueryConsumerChainStartProposalsResponse) {
+    option deprecated = true;
     option (google.api.http).get =
         "/interchain_security/ccv/provider/consumer_chain_start_proposals";
   }
 
+  rpc QueryConsumersThatAreAboutToStart(QueryConsumersThatAreAboutToStartRequest)
```

Comment:
Rename to `ConsumersAboutToStart`. Same for `QueryConsumersThatAreAboutToStop`.

Reply:
Also, let's not focus now on queries that are not needed for Forge.

Reply:
Ok, I removed the new queries.